src/features/surveys/utils/getSurveyUrl.ts

```diff
@@ -0,0 +1,11 @@
+import { ZetkinSurvey } from 'utils/types/zetkin';
+
+export default function getSurveyUrl(survey: ZetkinSurvey | null) {
```

Comment:
This is exactly the type of function where tests would be great. I don't think you need to add tests now, but it's one of those things that when you get proficient enough in writing tests, it's easier (at least for me) to create the function TDD than it is to develop it while clicking around the web app, with the added benefit that you can feel really confident about the logic.

Reply:
Agree! Making it a postit to stick on my computer - "Start with a test!"